src/inabox/amp-inabox.js

```diff
@@ -105,7 +105,7 @@ startupChunk(self.document, function initial() {
         installInaboxCidService(ampdoc);
         installViewerServiceForDoc(ampdoc);
         installInaboxViewportService(ampdoc);
-        installAmpdocServices(ampdoc, undefined, true);
+        installAmpdocServices(ampdoc, /* inabox */ true);
```

Comment:
seems you're on a pretty old master branch

Reply:
fixed